app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt

```diff
@@ -131,9 +131,9 @@ class AttendeeFragment : Fragment() {
         val privacyText = getString(R.string.privacy_text)
 
         paragraph.append(startText)
-        paragraph.append(termsText)
-        paragraph.append(middleText)
-        paragraph.append(privacyText)
+        paragraph.append(" "+ termsText)
+        paragraph.append(" "+ middleText)
+        paragraph.append(" "+ privacyText)
```

Comment:
you could write something like `" $privacyText"`

Reply:
> you could write something like `" $privacyText"`

Thanks! I changed it.